extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js

```diff
@@ -3,44 +3,87 @@ require([
 	'mw',
 	'wikia.loader',
 	'wikia.mustache',
-	'wikia.tracker'
-], function ($, mw, loader, mustache, tracker) {
+	'wikia.tracker',
+	'wikia.abTest'
+], function ($, mw, loader, mustache, tracker, abTest) {
 	'use strict';
 
 	var $banner,
 		dismissCookieName = 'pmp-entry-point-dismissed',
+		experimentName = 'POTENTIAL_MEMBER_PAGE_ENTRY_POINTS',
 		track = tracker.buildTrackingFunction({
 			category: 'potential-member-experiment',
 			trackingMethod: 'analytics'
-		});
+		}),
+		experiment,
+		experiments = {
+			TOP: function () {
+				this.type = 'top';
+				this.addEntryPoint = function () {
+					$banner.insertAfter($('#WikiaPageHeader .header-container'));
+				}
+			},
+			IN_ARTICLE: function () {
+				this.type = 'in-article';
+				this.addEntryPoint = function () {
+					var $content = $('.mw-content-text'),
```

Comment:
You can actually avoid using this since there's always an existing reference to it in `mw.util.$content` so don't need to select it again yourself.